index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="portfolio" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <div class="container">
```

Comment:
Is this div tag necessary?
I see no styles applied to this container, so I wondered if you were planning to use this class for later.

Reply:
Deleted it since it was not necessary.